Assets/Scripts/Player/Player.cs

```diff
@@ -61,13 +58,10 @@ public class Player : MonoBehaviour {
 			hand.Add(deck.DrawCard());
 		}
 
-		// Set all player's inactive cards to active
-		// for( card in cardsOnField ){
-		//    card.isActive = true;
-		// }
-
-		// Trigger any events
-		// whenTurnIsStarted
+		for(card in cardsOnField){
+		   card.isActive = true;   // Set all player's inactive cards to active
+				card.whenTurnIsStarted; // Trigger any events this card has on turn start
```

Comment:
This code doesn't do anything as written, but we don't have a mechanism yet to cause effects based on GameActions so don't worry about it for now

Reply:
Ahh true, shoulda commented that one line out ;)